examples/albert/run_trainer.py

```diff
@@ -165,11 +165,12 @@ def get_optimizer_and_scheduler(training_args, model):
 
 class CollaborativeCallback(transformers.TrainerCallback):
     def __init__(self, dht: hivemind.DHT, optimizer: hivemind.CollaborativeOptimizer,
-                 model: torch.nn.Module, trainer_uuid: str, statistics_expiration: float):
+                 model: torch.nn.Module, local_public_key: bytes, statistics_expiration: float):
```

Comment:
```suggestion
                 model: torch.nn.Module, public_key: bytes, statistics_expiration: float):
```

Reply:
In fact, I think it's safe to replace all occurrences of local_public_key with public_key as long as we're talking about attributes of objects (and arguments of their initialization methods)

Reply:
See the [discussion](https://github.com/learning-at-home/hivemind/pull/250#discussion_r625110139) below.